packages/starknet-snap/src/services/node/provider.ts

```diff
@@ -0,0 +1,53 @@
+import { Provider, RawCalldata, CallContractResponse, ProviderOptions } from 'starknet';
+
+import { Network } from '../../types/snapState';
+
+export interface NodeProvider {
```

Comment:
I think it's better to change the name of the interface because it's the same as the class here

Reply:
i will keep this name until i found 1 better, and it is okay in JS for the interface has same name with class